src/lib/eina/eina_prefix.c

```diff
@@ -42,8 +42,8 @@
 #endif
 
 #ifdef _WIN32
-# include <direct.h> /* getcwd */
 # include <evil_private.h> /* path_is_absolute realpath dladdr */
+# include <direct.h> /* getcwd */
```

Comment:
Are there any problems with the other order?

Reply:
I didn't test with this particular case but if `<direct.h>` includes `<windows.h>`, possibly it will  mess with our `WIN32_LEAN_AND_MEAN`, that is used by `<evil_private.h>` include.

Reply:
oh, okay